shared/reducers/favorite.js

```diff
@@ -19,6 +20,12 @@ export default function (state: State = initialState, action: FavoriteAction): S
         ...state,
         folders: action.payload && action.payload.folders
       }
+    case logoutDone: {
```

Comment:
we should discuss this. when you logout i'm clearing this but maybe we should really blow away most of the store when this happens. @MarcoPolo @cjb  thoughts?